src/index.html

```diff
@@ -7,6 +7,16 @@
   <link rel="icon" href="/logo.png">
   <link rel="manifest" href="/manifest.webmanifest">
   <title>Kanttiinit</title>
+
+  <!-- Facebook Meta tags -->
+  <meta property="og:url"                content="http://www.kanttiinit.fi" />
+  <meta property="og:type"               content="website" />
+  <meta property="og:title"              content="Kanttiinit" />
+  <meta property="og:description"        content="Student lunch effortlessly" />
```

Comment:
I think our official slogan should end with a period "Student lunch effortlessly."